src/test/java/uk/gov/register/service/ItemValidatorTest.java

```diff
@@ -151,4 +167,91 @@ public void validateItem_failsOnEmptyStringField() throws IOException {
             assertThat(e.getEntry().toString(), equalTo(jsonString));
         }
     }
+
+    @Test
+    public void validateItem_date_year() throws IOException {
```

Comment:
The general convention for our unit tests is something along the lines of:

`methodName_shouldDoSomething_whenGivenSomeInputDataOrSomeConditionIsTrue`

In the context of this test, for example:

`validateItem_shouldValidateSuccessfully_whenInputDateTimeIsOfFormatYYYYDD`